libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -9,6 +9,8 @@ import { StripeClient } from './stripe.client';
 import { STRIPE_MINIMUM_CHARGE_AMOUNTS } from './stripe.constants';
 import { CustomerDeletedError, SubscriptionStripeError } from './stripe.error';
 
+const MOZILLA_TAX_ID = 'Tax ID';
```

Comment:
This should move to the stripe.constants.ts file in this lib

Reply:
Updated!